capstone/src/main/webapp/scripts/auth.js

```diff
@@ -33,22 +33,27 @@ function authenticateUser() {
  .then(response => response.json())
  .then((login) => {
     // need to set button's url to login url.
-    const loginUrl = login.url;
-    const loginButton = document.getElementById('login-btn');
-    loginButton.setAttribute('href', loginUrl);
+    const loginUrl = login.loginurl;
+    const loginButtons = document.getElementsByClassName("login-btn");
+    for(let i = 0; i < loginButtons.length; i++) {
+      loginButtons[i].setAttribute('href', loginUrl);
+    }
+    const loginStatus = login.loggedIn;
+
+    if (loginStatus) {
+      signUpUser();
+    }
  });
 }
 
 /*
- * Sign up the user. Placeholder code.
+ * Sign up the user for the app if they have not already signed up.
  */
 function signUpUser() {
```

Comment:
rename this function to reflect that it is used for logged in and newly registered users.